definitions/reports/advisor_on_prem_remediations.rb

```diff
@@ -0,0 +1,31 @@
+module Reports
+  class AdvisorOnPremRemediations < ForemanMaintain::Report
+    metadata do
+      description 'Report number of remediations based on advisor rules from advisor on premise'
+      confine do
+        config_file = '/etc/foreman/plugins/foreman_rh_cloud.yaml'
+        setting_regex = /:use_local_advisor_engine:\s*true/
```

Comment:
Can you rely on the installer answers instead?

`foreman::plugin::rh_cloud::enable_iop_advisor_engine`